### ISSUE TITLE
Entries not appearing in UI

### ISSUE THREAD

**Issue by jloh** [sveltia/sveltia-cms#273]
Hey! Hope you're well. I'm having a weird experience with Sveltia v0.48.0 (though I've had this in older versions so I don't think its new) where new entries aren't visible after a refresh of the browser window. I'm using Firefox 133.0.3 (aarch64) on OSX 15.1.1 if that helps.

I can't reliably reproduce it but what happens is:

1. Create a new post either manually or through the Sveltia panel
2. If created through the panel see it in the post list
3. Refresh page, see that post isn't appearing anymore

I know thats not a lot to go on, sorry 😅

Things I've tried to debug:

- Turned on advanced features, no hints there
- No errors in browser console apart from one (I'll paste below)
- Deleted browser cache + Index DB to force it to pull down GitHub data again
- Do another manual commit to increase the commit hash
- Chrome and Safari, same problem unfortunately

When manually viewing the Index DB data I can see the post (or posts if theres more!) are visible in the DB, Sveltia isn't displaying them though for some reason.

I thought it might've been a dodgy timestamp but it appears to be fine so I'm a bit lost unfortunately. I know that isn't a lot to go on so let me know what I can do to help debug!

<details><summary>Config</summary>

```yaml
backend:
  branch: main
  name: github
  repo: jloh/blog
collections:
  - create: true
    fields:
      - default: "{{now}}"
        format: YYYY-MM-DDTHH:mm:ssZ
        label: Publish Date
        name: date
        picker_utc: false
        widget: datetime
      - label: Body
        name: body
        widget: markdown
      - default: []
        label: Tags
        label_singular: Tag
        name: tags
        required: false
        widget: list
      - default: true
        label: Draft
        name: draft
        widget: boolean
      - hint: External link used in feeds
        label: External URL
        name: link
        required: false
        widget: string
      - hint: URL to Mastodon post
        label: Mastodon URL
        name: posts.mastodon
        required: false
        widget: string
      - hint: URL to Bluesky post
        label: Bluesky URL
        name: posts.bluesky
        required: false
        widget: string
    folder: content/notes
    icon: note_stack
    label: Notes
    name: notes
    path: "{{year}}/{{slug}}"
    slug: "{{date | date('YYYY-MM-DD_HHmm')}}"
    summary: "{{draft}} | {{date | date('YYYY-MM-DD h:mm a')}} - {{body |
      truncate(60, '&hellip;')}}"
  - create: true
    fields:
      - label: Title
        name: title
        widget: string
      - label: Body
        name: body
        widget: markdown
      - default: "{{now}}"
        format: YYYY-MM-DDTHH:mm:ssZ
        label: Publish Date
        name: date
        picker_utc: true
        widget: datetime
      - default: []
        label: Tags
        label_singular: Tag
        name: tags
        required: false
        widget: list
      - default: true
        label: Draft
        name: draft
        widget: boolean
      - hint: The path under `/posts/` this is displayed at
        label: Slug
        name: slug
        required: false
        widget: string
    folder: content/posts
    label: Posts
    name: posts
    summary: "{{title}} - {{date | date('ll')}}"
    view_filters:
      - field: draft
        label: Drafts
        pattern: true
locale: en-AU
media_folder: assets/imgs
public_folder: /imgs
search: true
site_url: https://jloh.co/
```

</details>

<details><summary>Browser error</summary>

```
Uncaught TypeError: _y is not a function
    <anonymous> sveltia-cms.996265edecb98e5968c5b830d7f57d94b079e6ea4190a49876581815197abd7b3754635272059534fb9b600c43d00cdc298d016f1e22786a90bf7c00593c13e3.js:formatted:85841
    <anonymous> sveltia-cms.996265edecb98e5968c5b830d7f57d94b079e6ea4190a49876581815197abd7b3754635272059534fb9b600c43d00cdc298d016f1e22786a90bf7c00593c13e3.js:formatted:85843
sveltia-cms.996265edecb98e5968c5b830d7f57d94b079e6ea4190a49876581815197abd7b3754635272059534fb9b600c43d00cdc298d016f1e22786a90bf7c00593c13e3.js:formatted:85841:4
    <anonymous> sveltia-cms.996265edecb98e5968c5b830d7f57d94b079e6ea4190a49876581815197abd7b3754635272059534fb9b600c43d00cdc298d016f1e22786a90bf7c00593c13e3.js:formatted:85841
    <anonymous> sveltia-cms.996265edecb98e5968c5b830d7f57d94b079e6ea4190a49876581815197abd7b3754635272059534fb9b600c43d00cdc298d016f1e22786a90bf7c00593c13e3.js:formatted:85843
```

Actual error location:

```javascript
    window.CMS = _y;
    window.initCMS = wy;
    !window.CMS_MANUAL_INIT &&
    (
      document.currentScript ||
      document.querySelector('script[src$="/sveltia-cms.js"]')
    ) &&
    wy();
    window.CMS_MANUAL_INIT = !0;
    _y();
  }
) ();
```

Not sure whats happening there, I think because I'm manually init'ing it?

</details>

**Comment by kyoshino** [comment]
Will check.

**Comment by kyoshino** [comment]
Fixed in [v0.49.0](https://github.com/sveltia/sveltia-cms/releases/tag/v0.49.0) 🙇🏼
